### PR TITLE
update the rke provisioner

### PR DESCRIPTION
**Pull request by jacobweinstock**
update the RKE provisioner to create the permanent k8s nodes instead of a bootstrap VM

### CODE REVIEW

File: pkg/providers/vsphere/preparerke.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+func (v *MgmtBootstrapRKE) Prepare() error {
+	err := v.createFolders()
+	if err != nil {
+		return err
+	}
+	configYAML, err := yaml.Marshal(v)
+	if err != nil {
+		return err
+	}
+	prereqs := `curl https://get.docker.com/ | bash`
```
Comment (j-griffith) [inline pkg/providers/vsphere/preparerke.go:20]: There are a number of other steps required (kernel modules); but do we want to just expect all this as being "done" already and baked into our images?  If no, then we at least need to add some missing things eg: https://github.com/j-griffith/packer-rke/blob/master/scripts/rke-install.sh

Comment (jacobweinstock) [reply pkg/providers/vsphere/preparerke.go]: at least for now, I vote we don't expect these prereqs to be "done". I've updated the PR accordingly, please review.

**Comment by jacobweinstock** [comment]
> I think we should walk through this in our next stand-up, just to get everything synced up.

sounds good.

File: pkg/providers/vsphere/preparerke.go (hunk 1 above)
Comment (jacobweinstock) [reply pkg/providers/vsphere/preparerke.go]: after talking, I've removed the prereq stuff. we'll revisit it later.

**Pull request merged**